compute_endpoint/globus_compute_endpoint/endpoint/config/utils.py

```diff
@@ -230,24 +230,36 @@ def _shell_escape_filter(val):
     return json.dumps(shlex.quote(loaded))
 
 
-def render_config_user_template(endpoint_dir: pathlib.Path, user_opts: dict) -> str:
-    import jinja2  # Only load package when called by EP manager
+def load_user_config_template(endpoint_dir: pathlib.Path) -> tuple[str, dict | None]:
```

Comment:
This method needs MU-admin privileges to touch the user configs. It would be nice to clarify that this method should be called before dropping privileges.